app/src/main/res/values/strings.xml

```diff
@@ -243,6 +243,8 @@
     <string name="safe_settings_fallback_handler">Fallback handler</string>
     <string name="safe_settings_nonce">Nonce</string>
     <string name="safe_settings_modules">Addresses of enabled modules</string>
+    <string name="safe_settings_fallback_handler_help">What is a fallback handler and how does it relate to the Gnosis Safe?</string>
+    <string name="safe_settings_fallback_handler_help_url">https://help.gnosis-safe.io/en/articles/4738352-what-is-a-fallback-handler-and-how-does-it-relate-to-the-gnosis-safe</string>
```

Comment:
this could go to static strings

Reply:
I moved both urls. See: a2660a78fd7fbf9914474fcff36cc46429e12f58